readme.md

```diff
@@ -157,6 +158,7 @@ Options:
 | --filterFile         | -f            | the file to specify filter setting                                          | path to file | defaultFilter.json          | optional  |
 | --casingFile         | -c            | the file to specify casing setting                                          | path to file |                             | optional  |
 | --no-sort            |               | don't sort the OpenAPI file                                                 | boolean      | FALSE                       | optional  |
+| --keepEmptySchema    |               | don't remove empty schema object                                            | boolean      | FALSE                      | optional |
```

Comment:
very minor but could you align the table spacing at the end. It seems it is missing a space before "| optional |"

Reply:
nice catch ! it's fixed ;-)